### DISCUSSION TITLE
about conda environment

### DISCUSSION THREAD

**Discussion by EricGaoOm** [deepmodeling/dpgen#1606] [Q&A]
I came across the error "dargs.dargs.ArgumentKeyError: [at root location] key `fp_style` is required to choose variant but not found." in a very rapid frequency. My inference is some bugs happened in conda enviroments. I make multiple comparisons between key source code in the invalid envs and the worked one, and found no differences.
![图片1](https://github.com/user-attachments/assets/bb393189-2cfb-4778-bb7c-5d4df21bd0d2)

**Comment by njzjz** [comment]
Please check if `fp_style` is set.

**Reply by EricGaoOm** [reply]
In param.json the key value is set as **"fp_style": "vasp",**

**Reply by srgao-nnu** [reply]
Have you solved this problem? I have the same problem, is there any solution?
